code/src/index.css

```diff
@@ -1,13 +1,146 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+    margin: 0;
+    font-family: -apple-system, BlinkMacSystemFont, "Helvetica Neue",
+        sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    background-color: black;
+    color: white;
+    overflow-x: hidden;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.album-name a {
+    font-size: 14px;
+    font-weight: bold;
+}
+
+.artist-container {
+    font-size: 14px;
+    color: #a0a0a0;
+}
+
+.outer-wrapper {
+    display: grid;
+    grid-template-columns: auto;
+    width: 100vw;
+}
+
+header {
+    align-self: center;
+}
+
+.inner-wrapper {
+    width: 80vw;
+    max-width: 900px;
+    padding: 2%;
+    align-self: center;
+    justify-self: center;
+}
+
+.album-container {
+    display: grid;
+    grid-template-columns: auto;
+    row-gap: 5vh;
+}
+
+.album {
+    width: 300px;
+    height: auto;
+    display: grid;
+    grid-template-columns: auto;
+}
+
+.album img {
+    width: 100%;
+    height: auto;
+}
+
+a {
+    color: white;
+    cursor: pointer;
+    text-decoration: none;
+}
+
+.artist-container {
+    justify-self: center;
+    align-self: center;
+}
+
+.btn-container {
+    display: none;
+}
+
+/* TABLET */
+@media (min-width: 668px) and (max-width:1024px) {
+    .album-container {
+        grid-template-columns: auto auto;
+    }
+}
+
+/* DESKTOP */
+@media (min-width: 1025px) {
+    .album-container {
+        grid-template-columns: auto auto auto auto;
+    }
+
+    .album {
+        width: 210px;
+    }
+
+    a:hover {
+        text-decoration: underline;
+        font-weight: bold;
+        color: white;
+    }
+
+    .cover-and-btns {   
+        position: relative;
+    }
+    
+    .image-overlay {
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        color: transparent;
+        transition: 0.5s ease;
+    }
+    
+    .cover-and-btns:hover > .image-overlay {
+        background-color: rgba(0, 0, 0, 0.6);
+        transition: 0.5s ease;
+    }
+
+    .btn-container {
+        opacity: 0;
+        display: flex;
+        position: absolute;
+        justify-content: space-around;
+        top: 40%;
+        left: 4%;
+    }
+    
+    .btn {
+        display: block;
+        width: 21%;
+        background-color: rgba(0, 0, 0, 0);
+        filter: invert(1); 
+        border: none !important;
+        z-index: 1;
+    }
+
+    .play {
+        transform: scale(1.9);
+    }
+
+    .play:hover {
+        transform: scale(2.5);
+    }
+
+    .like:hover, .more:hover {
+        transform: scale(1.3);
+    }
+
+    .cover-and-btns:hover > .btn-container {
+        opacity: 1;        
+    }
```

Comment:
Nice bit of code that also provided some inspiration for us when we were struggling!